ossec_security/README.md

```diff
@@ -0,0 +1,204 @@
+
+## Overview
+
+[OSSEC][4] is an Open Source Host based Intrusion Detection System. It performs log analysis, integrity checking, Windows registry monitoring, rootkit detection, real-time alerting and active response. It helps to monitor and manage security events across various IT infrastructures.
+
+This integration ingests the following types of logs:
+- FTPD
+- Firewall
+- System
+- Syslog
+- SSHD
+- Pam
+- Windows
+- Web Access
+
+Visualize detailed insights into Firewall, FTPD, System, Syslog, SSHD, Pam, Windows, Web Access logs through the out-of-the-box dashboards.
+
+## Setup
+
+### Installation
+
+To install the OSSEC Security integration, run the following Agent installation command and the steps below. For more information, see the [Integration Management][6] documentation.
+
+**Note**: This step is not necessary for Agent version >= 7.54.0.
+
+Linux command
+  ```shell
+  sudo -u dd-agent -- datadog-agent integration install datadog-ossec_security==1.0.0
+  ```
+
+### Configuration
+
+#### Logs Collection
+
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in datadog.yaml:
+
+    ```shell
+    logs_enabled: true
+    ```
+2. Add this configuration block to your `ossec_security.d/conf.yaml` file to start collecting your logs.
+
+    We will be using the UDP method to collect the OSSEC alerts data.
+    See the [sample ossec_security.d/conf.yaml][8] for available configuration options.
+
+    ```yaml
+      logs:
+      - type: udp
+        port: <PORT>
+        source: ossec-security
+        service: ossec-security
+    ```
+    **Note**: It is recommended not to change the service and source values as these parameters are integral to the pipeline's operation.
+
+3. [Restart the Agent][3].
+
+#### Configure Syslog Message Forwarding from OSSEC.
+  1. Add following configuration in `/var/ossec/etc/ossec.conf`.
+
+      In this example all alerts are sent to 1.1.1.1 on port 8080 in **JSON** format.
+      ```xml
+        <syslog_output>
+          <server>1.1.1.1</server>
+          <port>8080</port>
+          <format>json</format>
+        </syslog_output>
+      ```
+
+      *IP_ADDRESS(1.1.1.1): ip address where your datadog-agent is running.
+
+      *PORT(8080): port on which your datadog-agent is listening.
+
+      Note: Using JSON format is required since OSSEC Security pipeline parses JSON formatted logs only.
+
+  2. After this change is made, the client-syslog process should be enabled:
+      ```shell
+      /var/ossec/bin/ossec-control enable client-syslog
+      ```
+
+  3. Restart OSSEC service:
+      ```shell
+      /var/ossec/bin/ossec-control restart
+      ```
+
+#### Firewall Logs Collection[Optional]:
+Since OSSEC server does not forward the firewall alert logs via syslog by default, if you want to forward firewall alert logs via OSSEC server follow the below step.
+
+  1. Locate `firewall_rules.xml` file.
+  ```
+  /var/ossec/rules/firewall_rules.xml
```

Comment:
I recommend removing this in favor of the edit above, we don't usually list paths as standalone code blocks.

Reply:
Updated as per suggestion